openwisp_ipam/tests/test_api.py

```diff
@@ -251,6 +251,24 @@ def test_hosts_list_api(self):
         self.assertEqual(self.client.get(response.data['previous']).status_code, 200)
         self.assertIsNone(response.data['next'])
 
+    def test_bearer_auth(self):
+        self._create_user(username='tester', password='tester')
+        params = {'username': 'tester', 'password': 'tester'}
+        url = reverse('users:user_auth_token')
+        r = self.client.post(url, params)
+        token = r.data
+        subnet = self._create_subnet(subnet='10.0.0.0/24')
+        self._create_ipaddress(ip_address='10.0.0.1', subnet=subnet)
+        post_data = self._post_data(subnet=str(subnet.id), description='Testing')
+        response = self.client.post(
+            reverse('ipam:request_ip', args=(subnet.id,)),
+            data=post_data,
+            content_type='application/json',
+            HTTP_AUTHORIZATION=f'Bearer {token}',
+        )
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.data['ip_address'], '10.0.0.2')
+
```

Comment:
login in back at this point by doing `self._login()` to ensure other tests don't fail